src/widgets/FormInputs/FormDropdown.js

```diff
@@ -44,13 +45,19 @@ export const FormDropdown = ({
   return (
     <FlexColumn flex={1}>
       <FormLabel value={label} isRequired={isRequired} />
-      <DropDown
-        style={{ width: null, flex: 1 }}
-        values={valueStrings}
-        selectedValue={value[optionKey]}
-        onValueChange={onValueChangeCallback}
-        isDisabled={isDisabled}
-      />
+      {options.length ? (
+        <DropDown
+          style={{ width: null, flex: 1 }}
+          values={valueStrings}
+          selectedValue={value[optionKey]}
+          onValueChange={onValueChangeCallback}
+          isDisabled={isDisabled}
+        />
+      ) : (
+        <View>
+          <Text>{generalStrings.no_options}</Text>
+        </View>
```

Comment:
just having the drop down menu replaced with "NO OPTIONS" string. Not memoizing this as its just not needed